package.json

```diff
@@ -30,12 +30,12 @@
     "topo": "2.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=10 <17"
+    "hapi": "16.x.x"
```

Comment:
Wasn't sure about this, guessed based on Eran's note about supporting v16 in the v17 release notes.

Reply:
We should move it back to how it was, but no problem!  My rationale is this: dropping support for hapi 10-15 would be a breaking change, and there's no real advantage to make that change.  In hodgepodge v3 we'll drop support for 10-16.